examples/swapi/src/Page.re

```diff
@@ -3,11 +3,12 @@ let component = ReasonReact.statelessComponent("Page");
 let make = _children => {
   ...component,
   render: _self =>
-    <div> 
-      <h1> (ReasonReact.stringToElement("Star Wars")) </h1>
-       <Persons /> 
-       <AddPerson/>
-       <GetPerson />
-       <DeletePerson />
-    </div>
-};
+    <div>
+      <h1> (ReasonReact.string("Star Wars")) </h1>
```

Comment:
Nitpick: `|> ReasonReact.string`

Reply:
I do either way. Just happened to already be like this in the previous version of the example code. What is the argument one way or the other? Interested actually. Thanks.

Reply:
Just for consistency across the code, nothing more 😇